src/fu-util.c

```diff
@@ -3480,6 +3528,27 @@ fu_util_security(FuUtilPrivate *priv, gchar **values, GError **error)
 		}
 	}
 
+	/* any things we can fix? */
+	for (guint j = 0; j < attrs->len; j++) {
+		FwupdSecurityAttr *attr = g_ptr_array_index(attrs, j);
+		if (!fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS) &&
+		    fwupd_security_attr_get_bios_attr_id(attr) != NULL &&
+		    fwupd_security_attr_get_bios_attr_current_value(attr) != NULL &&
+		    fwupd_security_attr_get_bios_attr_target_value(attr) != NULL) {
```

Comment:
Assuming https://github.com/fwupd/fwupd/pull/4897 goes in, I think there should be a library change to not set the target value if the attribute is read only.

Reply:
https://github.com/fwupd/fwupd/pull/4905